locales/da/filters.json

```diff
@@ -91,6 +91,10 @@
 		"filter.23.name": "AdGuard Ukrainsk filter",
 		"filter.23.description": "Filter, der muliggør adblocking på ukrainsksprogede websteder."
 	},
+	{
+		"filter.24.name": "AdGuard Quick Fixes-filter",
```

Comment:
Error?
```suggestion
		"filter.24.name": "AdGuard Quick Fixes filter",
```

Reply:
nope, a hyphen is needed here I guess

Reply:
why not used in similar phrases of this lang?